### PR TITLE
Hemraker update umbraconetcoreupdates

### PR DESCRIPTION
**Pull request by hemraker**
Added recent blog posts and videos to NETCore/Umbraco 9 updates

### CODE REVIEW

**Comment by busrasengul** [comment]
Thank you for this @hemraker 
It all looks good to me!
Lets merge this in 🙌

**Comment by hemraker** [comment]
Nice! H5YR 🙌

File: UmbracoNetCoreUpdates.md [hunk 1]
```diff
@@ -50,14 +52,31 @@ In this section you will find a list of Umbraco .Net Core resources provided by
...
 * [Rick Butterfield: Umbraco Unicore first impressions](https://rickbutterfield.com/blog/umbraco-unicore-impressions)
 * [Greystate: Trying Out the .NET Core Umbraco Alpha Release](https://greystate.dk/log/2020/09/04/umbraco-net-core-alpha/)
 
 ### Other
 
+* [Github repository: Umbraco 9 Demo site ](https://github.com/Adolfi/UmbracoNineDemoSite) 
+* [YouTube: Porting an Umbraco package to v9 using multi-targeting - The Package Team](https://www.youtube.com/watch?v=W9IvX9MmoMo)
+* [YouTube: How to Host an Umbraco 9 site with IIS](https://www.youtube.com/watch?v=riO5EQKE0YY)
+* [YouTube: Run .NETCore CMS Umbraco on a MacBook](https://www.youtube.com/watch?v=lDL9tsJojfs)
+* [YouTube: Migrating Enent Handlers in Umbraco V9](https://www.youtube.com/watch?v=o_rpltxwj-8)
```
Comment (Jeavon) [inline UmbracoNetCoreUpdates.md:78]: typo - "Enent"

Comment (busrasengul) [reply UmbracoNetCoreUpdates.md:78]: Oh missed that! Fixing it now 👍

**Pull request merged**